lib/Core.lua

```diff
@@ -63,6 +63,14 @@ function Core.createElement(elementType, props, children)
 	end
 
 	props = props or {}
+	
+	if props.Name then
+		warn("Name prop is unused. ")
```

Comment:
It'd be nice to explain *why* these props shouldn't be specified, and suggest alternatives. It might also be useful to specify a stack trace so that people can quickly find the source of the warning!

Reply:
I have no experience with stacktraces. I’ll try to play around with those.